packages/discord.js/src/util/Events.js

```diff
@@ -148,12 +143,6 @@ module.exports = {
   MessageReactionRemoveEmoji: 'messageReactionRemoveEmoji',
   MessageUpdate: 'messageUpdate',
   PresenceUpdate: 'presenceUpdate',
-  Raw: 'raw',
-  ShardDisconnect: 'shardDisconnect',
-  ShardError: 'shardError',
-  ShardReady: 'shardReady',
-  ShardReconnecting: 'shardReconnecting',
-  ShardResume: 'shardResume',
```

Comment:
Why are we removing some of these? We can still emit them from ws, no?

Reply:
Use `client.ws.on('resume')` (etc) instead, no real reason to re-emit here.

Reply:
True